src/ILCompiler/repro/Program.cs

```diff
@@ -3,11 +3,346 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Text;
```

Comment:
Please no changes to the repro project. You can probably delete the `no_cpp` file under `tests\src\Simple\Interfaces` to unblock the tests in the CI.